---
int8 STATUS_OK=0
int8 STATUS_LOST_FIX=1
int8 STATUS_LOST_UTM_ZONE_BORDER=2
int8 STATUS_INVALID_FILE=3

int8 status
string waypoints_data
